vis/xAPISG-ItemsInteracted.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "outputs": [],
+   "source": [
+    "%run vis/xAPISG-noDataToFillVisualization.ipynb # notebook to create the visualisation with a message NoDataToFill\n",
+    "\n",
+    "def vis_items_interacted_times_interact():\n",
+    "    md_text = widgets.HTML(\"MultiBar Item Interaction\")\n",
+    "    multiBar=interactive(vis_items_interacted_times, order=order)\n",
+    "    return widgets.VBox([md_text, multiBar])\n",
+    "\n",
+    "def vis_items_interacted_times_heatmap_interacted():\n",
+    "    md_text = widgets.HTML(\"Heatmap Item Interaction\")\n",
+    "    heatMap=interactive(vis_items_interacted_times_heatMap)\n",
+    "    return widgets.VBox([md_text, heatMap])\n",
+    "\n",
+    "def vis_items_interacted_fonction_time_interacted():\n",
+    "    md_text = widgets.HTML(\"Bubble chart for item interaction in function of time\")\n",
+    "    bubbleChart=interactive(vis_items_interacted_fonction_time)\n",
+    "    return widgets.VBox([md_text, bubbleChart])"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "***vis_items_interacted_times* function**:\n",
+    "* Input: players_info dictionary\n",
+    "* Displays bar chart showing for each item the number of use by players"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "%run vis/HelpersFunctions/MultiBarSeparated.ipynb\n",
```

Comment:
Raises error "ERROR:root:File `'vis/HelpersFunctions/MultiBarSeparated.ipynb.py'` not found." when running analysis on xAPISGProcessor notebook.